test/wdio/package.json

```diff
@@ -3,7 +3,9 @@
   "type": "module",
   "version": "0.0.0",
   "scripts": {
-    "build": "node ../../bin/stencil build --debug --es5",
+    "build": "run-s build.main build.global-script",
+    "build.main": "node ../../bin/stencil build --debug --es5",
+    "build.global-script": "node ../../bin/stencil build --debug --es5 --config global-script.stencil.config.ts",
```

Comment:
expanded the `build` script here by moving the existing build to `build.main` and then adding a new one `build.global-script` - I think this can be a decent pattern to follow for the other similar tests